samples/diagnostics/program-visualization/TraceExample.ipynb

```diff
@@ -0,0 +1,553 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Using the `%trace` magic command to visualize your Q# operations\n",
+    "\n",
+    "In quantum computing literature, it is common to visualize quantum algorithms as quantum circuits. For example, when learning about [Simon's algorithm](https://en.wikipedia.org/wiki/Simon%27s_problem#Simon's_quantum_circuit) or [Quantum Teleportation](https://techcommunity.microsoft.com/t5/educator-developer-blog/quantum-teleportation-in-q/ba-p/380602) for the first time, it is useful to visualize them as a quantum circuit.\n",
```

Comment:
Makes sense, but it may be helpful to explain the difference here between the two approaches.

Reply:
Linked to "Limitations" section below.